docs/architecture/input-parameters-on-runtime.md

```diff
@@ -0,0 +1,23 @@
+# Input parameters for API, Event Definitions or API Packages in the Runtime
+
+## Overview
+
+Previously, in Kyma Runtime, while provisioning Service Class created from API/Event Definition, there were no possibility to pass input parameters. The ability may be useful in some specific use cases. This document describes passing input parameters from Kyma Runtime to Application or Integration System.
+
+## Solution
+
+The solution utilizes labels on Application level. User provides JSON schema for Service Class `instanceCreateParameterSchema` parameter by creating new Label Definition in the following format:
+
+ **Key**: 
+- For API Definition: `instance-create-params.apis.{api-def-name}`
```

Comment:
take into account proposal from integrationsystemName/input-param. Maybe we should have a convention for that.